src/Microsoft.AspNetCore.Sockets.Http/HttpConnectionDispatcher.cs

```diff
@@ -63,6 +58,31 @@ public HttpConnectionDispatcher(ConnectionManager manager, ILoggerFactory logger
             }
         }
 
+        public async Task ExecuteNegotiateAsync(HttpContext context, HttpSocketOptions options)
+        {
+            // Create the log scope and attempt to pass the Connection ID to it so as many logs as possible contain
+            // the Connection ID metadata. If this is the negotiate request then the Connection ID for the scope will
```

Comment:
This comment is a little different in this method since we know it definitely **is** the negotiate response :)